packages/office-ui-fabric-react/src/components/Image/Image.Props.ts

```diff
@@ -22,6 +22,11 @@ export interface IImageProps extends React.HTMLProps<HTMLImageElement>  {
    * Image source to display if an error occurs loading the image indicated by src.
    */
   errorSrc?: string;
+
+  /**
+   * If true, the image frame will expand to fill its parent container.
+   */
+  shouldMaximizeFrame?: boolean;
```

Comment:
Sounds good! Yeah, I know the props and classes are getting confusing as sometimes we're talking about the "frame" and sometimes the "image".